wp-content/themes/poetiktheme/front-page.php

```diff
@@ -324,175 +331,182 @@
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/staring.jpeg" alt="staring"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/flying.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/flying.jpeg" alt="flying"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/smiling.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/smiling.jpeg" alt="smiling"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/serious.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/serious.jpeg" alt="serious"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/bridge.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/bridge.jpeg" alt="bridge"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/live.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/live.jpeg" alt="live"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/staring.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/staring.jpeg" alt="staring"> -->
+                </div>
             </div>
         </div>
     </section>
 
-    <section class="shop ">
-        <h2 class="shop__title">photos</h2>
-
-        <div class="shop__slider">
-        </div>
-
-        <div class="shop__item-price">
-            <p><span>price: </span>21.89</p>
-        </div>
-
-        <button class="btn btn__buy">BUY</button>
-
-        <div class="shop__grid">
-            <div class="shop__card-wrapper">
-                <div class="shop__card">
-                    <div class="shop__item">
-                        <img src="" alt="">
-                    </div>
+    <section class="shop">
+        <div class="wrapper">
+            <h2 class="title">shop</h2>
+            <div class="shop__small-screen">
+                <div class="shop__slider">
+                </div>
+                <div class="shop__item-price">
+                    <p>price: 21.89</p>
+                </div>
 
-                    <div class="shop__item-message">
-                        <p>sold out</p>
+                <button class="btn btn__buy">BUY</button>
```

Comment:
Hummm... take a quick look at this document :  [https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99#:~:text=Responses%20(1)-,Why%20you%20shouldn't%20write%20your,in%20Uppercase%2C%20instead%20use%20CSS.&text=With%20that%20in%20mind%2C%20if,the%20text%2Dtransform%20CSS%20property.](url)

Reply:
It gives 404 page....but yeah I will change it to lower case